addons/sale_timesheet/models/project.py

```diff
@@ -526,7 +526,7 @@ def _get_default_partner_id(self, project, parent):
     is_project_map_empty = fields.Boolean("Is Project map empty", compute='_compute_is_project_map_empty')
     has_multi_sol = fields.Boolean(compute='_compute_has_multi_sol', compute_sudo=True)
     timesheet_product_id = fields.Many2one(related="project_id.timesheet_product_id")
-    remaining_hours_so = fields.Float('Remaining Hours on SO', compute='_compute_remaining_hours_so', search='_search_remaining_hours_so', compute_sudo=True)
+    remaining_hours_so = fields.Float('Remaining Hours on SO', compute='_compute_remaining_hours_so', search='_search_remaining_hours_so', compute_sudo=True, store=True)
```

Comment:
Can you revert that changes? We will no longer display the total of that field as explained in the chatter of your task. 🙂 

Thanks in advance!